Sources/Plasma/CoreLib/hsBitVector.h

```diff
@@ -116,137 +116,114 @@ class hsBitVector {
 
 inline hsBitVector::hsBitVector(const hsBitVector& other)
 {
-    if( 0 != (fNumBitVectors = other.fNumBitVectors) )
-    {       
+    if ((fNumBitVectors = other.fNumBitVectors)) {
```

Comment:
Personally I prefer to keep the extra comparison when an assignment is in an if statement...  Or else to separate the assignment out of the comparison, so it's clear the assignment is not a typo.

Reply:
I found this line to be confusing to begin with, so that makes sense. Will fix.